app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,11 @@
   </head>
 
   <body>
+    <header>
+      <span id="title">Fenton's Mini Shop</span>
+      <a href="/merchants" id="header-link">Merchants</a>
+      <a href="/items" id="header-link">Items</a>
+    </header>
```

Comment:
Good use of your application.html.erb file... adding the links that will persist throughout your whole app.